engine/installation/linux/docker-ee/centos.md

```diff
@@ -14,20 +14,22 @@ title: Get Docker EE for CentOS
 
 {% include ee-linux-install-reuse.md section="ee-install-intro" %}
 
-## Prerequisites
+## Install Docker EE on CentOS
+
+### Prerequisites
 
 Docker CE users should go to
 [Get docker CE for CentOS](https://docs.docker.com/engine/installation/linux/docker-ce/centos/)
 **instead of this topic**.
 
-### Docker EE repository URL
+#### Docker EE repository URL
 
 {% include ee-linux-install-reuse.md section="ee-url-intro" %}
 
-### OS requirements
+#### OS requirements
 
-To install Docker EE, you need a maintained version of CentOS 7. Archived
-versions aren't supported or tested.
+To install Docker EE, you need a host running a non-archived version of CentOS 7. Archived
```

Comment:
I think this is less clear. It's not clear that it is CentOS doing the archiving.